notebooks/vertex_matching_engine/solutions/matching_engine_ann.ipynb

```diff
@@ -0,0 +1,646 @@
+{
```

Comment:
<ul><li>"distance. (defined" -&gt; "distance (defined"</li><li>what does "AH" in tree-AH algorithm" mean? Approximative Hashing? Maybe spell it out. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/230/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>